catalyst/core/runner.py

```diff
@@ -117,6 +103,10 @@ def device(self, value: Device):
                 self._model, "to", device=self._device
             )
 
+    def _init(self):
+        self.experiment: _Experiment = None
+        self.state: _State = None
+
```

Comment:
Maybe we should remove type hints here, because the descendants have exactly the same `_init` except for type hints. Do you think they (type hints) really do matter here?

Reply:
for easier development, it's better to have type hints
just because the have no additional computational overhead during training this trick looks okay :)